site/docs/guide/files.md

```diff
@@ -32,8 +32,8 @@ You cannot reliably use a `file_id` from your friend's bot, to access a file wit
 Each bot will use different identifiers for the same file.
 This implies that you cannot simply guess a `file_id` and access some random person's file, because Telegram keeps track of which `file_id`s are valid for your bot.
 
-::: warning Using Foreign file_ids
-Note that in some cases it _is_ technically possible that a `file_id` from another bot seems to work correctly.
+::: warning Using Foreign file*ids
+Note that in some cases it \_is* technically possible that a `file_id` from another bot seems to work correctly.
```

Comment:
What does this mean? Maybe it's better to replace it with 
```suggestion
::: warning Using Foreign file\*ids
Note that in some cases it \_is\* technically possible that a `file_id` from another bot seems to work correctly.
```
so that the formatter doesn't get annoyed?

Reply:
Whoopsie

Reply:
The correct text would be this:
```suggestion
::: warning Using Foreign `file_id`s
Note that in some cases it *is* technically possible that a `file_id` from another bot seems to work correctly.
```
Does the formatter still mess that up?

Reply:
It might not cope with `mono` in the warning header very well